src/plugins/events/components/EventTimelineView.vue

```diff
@@ -196,16 +196,15 @@ export default {
       }
     },
     getClientWidth() {
-      let clientWidth = this.$refs.events.clientWidth;
-
+      // Try to use the component’s own element first
+      let clientWidth = this.$refs.events?.clientWidth;
       if (!clientWidth) {
-        //this is a hack - need a better way to find the parent of this component
-        const parent = this.openmct.layout.$refs.browseObject.$el;
+        // Fallback: use the actual container element (the immediate parent)
```

Comment:
I vaguely recall `.parentElement` not working in some specific cases for planView, but maybe that's not an issue here. I'll test.